pallets/parachain-staking/src/lib.rs

```diff
@@ -284,20 +284,20 @@ pub mod pallet {
 				None
 			}
 		}
-		// Returns None if nomination not found
-		pub fn inc_nomination(&mut self, collator: AccountId, more: Balance) -> Option<Balance> {
+		// Return false if nomination not found
+		pub fn inc_nomination(&mut self, collator: AccountId, more: Balance) -> bool {
```

Comment:
Maybe return `Result<()>`?

Reply:
What is your default error type here?

You'd have to make it generic over the `T: Config` to return a `Error<T: Config>` from any of these methods so I find it easier to just return `bool` and then propagate the relevant error in the runtime method logic.